Reply/app/src/main/java/com/example/reply/ui/navigation/ReplyNavigationActions.kt

```diff
@@ -69,4 +71,7 @@ class ReplyNavigationActions(navController: NavHostController) {
             restoreState = true
         }
     }
+    val navigateToDetail: (emailId: Long) -> Unit = { emailId ->
+     //   navController.navigate("${ReplyRouts.INBOX_ROUTE}/$emailId") {}
```

Comment:
I think this `navigateToDetail` needs to be updated. Right now, clicking on an item when just the list is showing doesn't cause anything to happen.

Reply:
ok this was redundant now, removed it.